index.html

```diff
@@ -371,14 +371,14 @@ <h3>
 decentralized digital identity. These new identifiers are designed to enable
 the controller of a DID to prove control over it and to be implemented
 independently of any centralized registry, identity provider, or certificate
-authority. DIDs are URLs that relate a DID subject to means for
-trustable interactions with that subject. DIDs resolve to DID Documents &mdash;
-simple documents that describe how to use that specific DID. Each DID Document
-may express cryptographic material, verification methods, and/or service
-endpoints. These provide a set of mechanisms which enable a DID controller to
-prove control of the DID. Service endpoints enable trusted interactions with the
-DID subject. This document specifies a common data model, format, and operations
-that all DIDs support.
+authority. DIDs are URLs that relate a DID subject to one or more mechanisms 
+that may be used for trustable interactions with that subject. DIDs resolve 
+to DID Documents, which are simple documents that describe how to use that 
+specific DID. Each DID Document may express cryptographic material, 
+verification methods, and/or service endpoints. These provide a set of 
+mechanisms which enable a DID controller to prove control of the DID. Service 
+endpoints enable trusted interactions with the DID subject. This document 
+specifies a common data model, format, and operations that all DIDs support.
```

Comment:
I'm struggling to find the rewording of this sentence which makes it flow cleanly, and have plural/singular agreement throughout -- `This document specifies a common data model, format, and operations that all DIDs support.`

Perhaps `This document specifies a common data model, a URL format, and a set of operations that all DIDs support.`